src/language-js/parse/postprocess/index.js

```diff
@@ -154,6 +157,29 @@ function postprocess(ast, options) {
     }
   });
 
+  /* eslint-disable prettier-internal-rules/no-node-comments */
+  if (isNonEmptyArray(ast.comments)) {
+    for (let i = ast.comments.length - 2; i >= 0; i--) {
+      const comment = ast.comments[i];
+      const nextComment = ast.comments[i + 1];
+      if (
+        isBlockComment(comment) &&
+        isIndentableBlockComment(comment) &&
+        isBlockComment(nextComment) &&
+        isIndentableBlockComment(nextComment) &&
+        locEnd(comment) === locStart(nextComment)
```

Comment:
`loc` compare should be faster than `isIndentableBlockComment` check, let's move this condition up